codec-agnostic-rtp-payload-format.md

```diff
@@ -220,6 +291,13 @@ a=fmtp:101 apt=98
 
 Q: What frequency should we use for audio? One per the different frequencies of each audio codec?
 
+Potential Alternatives
```

Comment:
I would not speak about the alternatives on the draft, we will en up having to specify all of them in detail and will only cause noise in the discussion.

Reply:
Let's remove it in this PR for now.

Reply:
Done